### PR TITLE
Re fetch

### PR DESCRIPTION
**Pull request by joonalillfors**
Dev dashboard fetches instances from server on instance removal instead of just filtering in frontend

## Review Checklist

- [x] All changes are described in the PR description
- [x] New feature and breaking changes have documentation
- [x] New endpoints have API tests
- [x] New frontend features have Enzyme tests
- [x] No unnecessary or debugging code
- [x] Questionable code is clearly marked with comments
- [x] Manually tested with latest Firefox, Safari and Chrome

### CODE REVIEW

**Comment by HenrikHelenius** [comment]
what about tests?

**Comment by joonalillfors** [comment]
> what about tests?

not a new feature

File: .gitignore [hunk 1]
```diff
@@ -28,3 +28,6 @@ proxy/proxy
 
 # Vim binaries on mac
 *.swp
+
+# Local config
+local.json
```
Comment (tobbe-j) [inline .gitignore:33]: Why is this included in this pr? Not described in pr description

**Review by tobbe-j — APPROVED** [review]
Looks good

File: .gitignore (hunk 1 above)
Comment (joonalillfors) [reply .gitignore:33]: It's from slackbot branch and I accidentally added the local.json to this branch

**Review by kalomak — APPROVED** [review]
Ignore gitignore

File: .gitignore (hunk 1 above)
Comment (kalomak) [reply .gitignore:33]: Idiot
